onchain/src/social_account.cairo

```diff
@@ -0,0 +1,20 @@
+#[starknet::contract]
+pub mod SocialAccount {
+    #[storage]
+    struct Storage {
+        public_key: felt252,
+    }
+
+    #[event]
+    #[derive(Drop, starknet::Event)]
+    enum Event {
+        AccountCreated: AccountCreated,
```

Comment:
Is this finished?

Reply:
Yes, are we adding anything aside the event name